addons/hr_expense/static/src/components/qrcode_action.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<templates>
+    <t t-name="hr_expense.QRModalComponent">
```

Comment:
Before Odoo 17, there was an extra attribute to determine which rendering engine to use, or something like that:
```suggestion
    <t t-name="hr_expense.QRModalComponent" owl="1">
```

Reply:
Fixed!